modules/rest-api/pages/upload-retrieve-node-cert.adoc

```diff
@@ -85,14 +85,16 @@ A JSON object of the following kind registers a passphrase of type `rest`:
                           "url": "<url to call>",
                           "addressFamily": inet6,
                           "httpsOpts": {"verifyPeer": true}
+                          "headers": {"key1": "value1", "key2": "value2"}
                           "timeout": 5000}}
 ----
 
 The specified `type` is `rest`.
-The value of `url` must be the endpoint to be called with `GET`.
-The `addressFamily` specifies the address family to be used, and can be either `inet4` or `inet6`.
-The value of `httpsOpts` must be a JSON key-value pair whose key is `verifyPeer`, and whose value is either `true` or `false`: this determines whether peer verification is performed if Https is used.
-The value of `timeout` specifies the number of milliseconds that must elapse before the call is timed out.
+The value of `url`, which is a _required_ argument, must be the endpoint to be called with `GET`: the endpoint can use either `http` or `https`.
+The `addressFamily`, which is an _optional_ argument, specifies the address family to be used, and can be either `inet` (meaning IPv4) or `inet6` (meaning IPv6): the default value is `inet`.
+The value of `httpsOpts`, which is an _optional_ argument that is valid only when `https` is used in the value specified for `url`, must be a JSON key-value pair whose key is `verifyPeer`, and whose value is `true`: this determines that peer verification is performed when Https is used &#8212; `true` is the only value supported, and is the default.
```

Comment:
verifyPeer is the only option supported in httpsOpts
verifyPeer can be either true or false, true by default

Reply:
Thanks, I have rewritten.